addons/containerd/1.6.20/kubeadm-join-config.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: kubeadm.k8s.io/v1beta2
+apiVersion: kubeadm.k8s.io/$(kubeadm_conf_api_version)
```

Comment:
can you explain to me how the function invocation in this file is evaluated? it is unclear to me.

Reply:
I'm actually thinking of undoing this change and instead replace the api version of the addon patches at runtime. None of the current kubeadm conf addon patches use `v1beta2` specific fields so I think this is safe to do.

Reply:
`$(kubeadm_conf_api_version)` will be evaluated when the final kubeadm config file rendered: https://github.com/replicatedhq/kURL/blob/main/scripts/install.sh#L195